lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2707,6 +2708,299 @@ def calc_arrow(uvw, angle=15):
 
     quiver3D = quiver
 
+    def errorbar(self, x, y, z, zerr=None, yerr=None, xerr=None, fmt='',
+                 barsabove=False, errorevery=1, ecolor=None, elinewidth=None,
+                 capsize=None, capthick=None, xlolims=False, xuplims=False,
+                 ylolims=False, yuplims=False, zlolims=False, zuplims=False,
+                 **kwargs):
+        """
+        Draws error bars on an Axis3D instance.
+
+        Parameters
+        ----------
+        x : scalar
+        y : scalar
+        z : scalar
+
+        xerr/yerr/zerr : scalar or array-like, shape(n,1) or shape(2,n).
```

Comment:
`err`-like arguments are optional to allow flexibility in choosing the dimension your errors are plotten in. For example, one can specify `xerr` and `zerr` to be `None`, so that only y-axis errors would be plotted. Therefore, any given argument from the `xerr/yerr/zerr` can be optional.